storagemarket/impl/providerstates/provider_states_test.go

```diff
@@ -277,6 +271,38 @@ func TestPublishDeal(t *testing.T) {
 				require.Equal(t, "error calling node: ensuring funds: not enough funds", deal.Message)
 			},
 		},
+	}
+	for test, data := range tests {
```

Comment:
Test of case where provider gets MCID from node, goes to StorageDealProviderFunding?